Resources/Private/JavaScript/media-module/src/components/Dialogs/UploadDialogs/Presentationals/FilePreview.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
```

Comment:
I wasn't sure with the folder structure but these presentationals, are only reusable in an upload dialog, so I gave them their own subfolder

Reply:
They are not fully `presentationals` IMO as they have some internal dependencies. 
I refactored the classes into their own suppackge. So that's fine now.